### PR TITLE
Upgrade to Rector 0.10

### PR DESCRIPTION
**Pull request by TomasVotruba**
## Benefits from Rector 0.10

- https://getrector.org/blog/2021/03/22/rector-010-released-with-php71-support
- https://getrector.org/blog/2021/03/15/legacy-refactoring-made-easy-with-static-reflection
- https://getrector.org/blog/from-doctrine-annotations-parser-to-static-reflection

### CODE REVIEW

**Comment by TomasVotruba** [comment]
The `build_scoped_typo3_rector` should run only on `master` branch (we do it this way Rector) or for contributors with access. It will always fail for a PR from non-owner

**Comment by TomasVotruba** [comment]
Ready to merge :heavy_check_mark: 


Phew :)

File: src/Console/Application.php [hunk 1]
```diff
@@ -8,32 +8,27 @@
...
 use Symfony\Component\Console\Application as SymfonyApplication;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputDefinition;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symplify\PackageBuilder\Console\Command\CommandNaming;
-use Throwable;
 
+/**
```
Comment (sabbelasichon) [inline src/Console/Application.php:22]: @TomasVotruba It was only for the stubs as i remember correctly. But now it is gone and we could use the one from Rector. Hooray.

**Comment by sabbelasichon** [comment]
@TomasVotruba Thank you very much, this is really big step forward. Amazing work.

**Pull request merged**